remote.go

```diff
@@ -263,7 +218,7 @@ func printRemoteDetails(remote *client.RemoteVPS) {
 func init() {
 	rootCmd.AddCommand(remoteCmd)
 	remoteCmd.AddCommand(addCmd)
-	remoteCmd.AddCommand(statusCmd)
+	//remoteCmd.AddCommand(statusCmd)
```

Comment:
Same here - if we don't need code anymore, just delete it. We can see the history in the git history, so don't worry about losing anything! 📖 

Leaving commented code, especially without an explanation, usually means no one will ever remove it (out of fear it might actually be important), and eventually it just gathers up all over the place 😢